File: Difficulty: Medium/Pattern/pattern.java
// User function Template for Java

class Solution {

    void printDiamond(int n) {
        // Your code here
          for(int i=0;i<n;i++){
              for(int space=0;space<n-i-1;space++){
                  System.out.print(" ");
              }
              for(int j=0;j<=i;j++){
                  System.out.print("* ");
              }
              System.out.println();
          }
          for(int i=0;i<n;i++){
              for(int space=0;space<i;space++){
                  System.out.print(" ");
              }
              for(int j=0;j<n-i;j++){
                  System.out.print("* ");
              }
              System.out.println();
          } 
    }
}

File: Difficulty: Medium/Pattern/pattern.py
#User function Template for python3

class Solution:
    def printDiamond(self, N):
        # Code here
        for i in range(1,N+1):
            for space in range(N-i):
                print(" ",end="")
            for j in range(i):
                print("*",end=" ")
            print()
            
        for i in range(1,N+1):
            for space in range(i-1):
                print(" ",end="")
            for j in range(N-i+1):
                print("*",end=" ")
            print()